eslint.config.mjs

```diff
@@ -50,14 +58,23 @@ export default [
     rules: {
       ...ts.configs['eslint-recommended'].rules,
       ...ts.configs.recommended.rules,
-      'no-undef': 'off' // redundant - TS will fail to compile with undefined vars
+      'no-undef': 'off', // redundant - TS will fail to compile with undefined vars
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        { ignoreRestSiblings: true, argsIgnorePattern: '^_', destructuredArrayIgnorePattern: '^_' }
```

Comment:
are there cases where we want to keep the `_var` format for specific vars? I'd prefer to use the `after-used` option here personally: https://eslint.org/docs/latest/rules/no-unused-vars#args

Reply:
Personally I'm not a fan of comma dangle outside of this use case as it's usually just a crutch to avoid having to name things.  Note that I didn't specify `varsIgnorePattern` here.  Updated to use `after-used`